### PR TITLE
Apple background and gradient project

### PR DESCRIPTION
**Pull request by marcelomaidden**
## Apple background and gradient project
In this project we used:

- Flexbox and grid to create the layout
- Float to create the footer section
- background-image with linear-gradient on the menu
- Navigation bar with border-radius

### CODE REVIEW

File: stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+}
+
+.border-l:hover {
+  border-radius: 10px 0 0 10px;
+}
+
+nav li:hover {
+  background-image: linear-gradient(darkgrey, black);
+
+  /* border-radius: 10px; */
```
Comment (ghost) [inline stylesheets/style.css]: Remove the comment on line 83.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Good job on your project. However, there are some few changes needed:**

1). **Reduce the extra width after the navbar search input, so it matches the original.**

original  |  clone
--------- |  ---------
gradient_original |  gradient_clone

2). **The main content's background image doesn't cover the full-screen size. There is some white space at the rightmost part of the image.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from pathwright on requirements).**

background_clone
 
3). **Remove the horizontal scroll.**

scroll

**Check for one more comment below.**

## Best of luck 😃!

**PS: You can send me a message on slack(Rose Wanjohi)**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @imran5644 and @marcelomaidden
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/94326828-4387d300-ffc0-11ea-865c-55e523f49f0a.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/)


### Header Background image
- [ ] Header background image is not covering the full screen (Previous TSE explained it)

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[Clone Page](https://rawcdn.githack.com/imran5644/apple_backgrounds_and_gradients/c23bd270138415152478e46f5d12791e02b7f8d0/index.html)|
|:---:|:---:|
|source-page|clone|



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team! Good job with this project, look pretty good! 👍 
Good job applying the changes 💯 
However, still one more change to apply before being approved! 💪 

Please, make the changes and resubmit your project for a new code review, so you can move into the next project ⏩ 

Check my comments below 👇 and good luck 😃 

- Your live link is not updated, so on your web, I still can see the horizontal scrollbar and some problems with the sizing, I had to create my own live link to check your changes, please update it, it is important that your live link works properly, just think it is the first thing the recruiters look at.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great work!

:100: nice implementing media queries even if the project didn't require it! :clap: :clap:

Great effort working on the previously requested changes. There are still, some issues to be addressed.

## Required Changes
### [Mandatory] :red_circle: 

- [ ] Feature branch should not be defaul branch.
image

- [ ] Footer not aligned properly
image

- [ ] Remove duplicated Font-Awesome link from the head. See inline comments

- [ ] Remove unnecessary font Roboto link from the head. See inline comments

- [ ] Remove ```<div>``` container wrappers from block elements. See inline comments.

### [OPTIONAL]  :large_blue_circle: 

- [ ] Use 2 space indentation in HTML file. This helps improve Readability.

- [ ] ```<header>```, ```<main>```, and ```<footer>``` should be the most outer tags inside the ```<body>```. The background image for the hero section could be approached in another way.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        <p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.
+                        </p>
+                    </div>
+    
+                    <div>
+                        <p>
+                            <a href = "#">Watch the “Your Verse” film and explore the stories</a> 
+                            <i class="fa fa-play-circle watch-icon"></i>
+                        </p>
+                    </div>
```
Comment (jubaan) [inline index.html]: - [ ] ```<h>```, and ```<p>``` are HTML block elements (not inline), using a div to wrap them is redundant and unnecessary.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <title>
+            Apple Store
+        </title>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <meta charset="UTF-8">
+            <link rel="stylesheet" type="text/css" href="stylesheets/style.css">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+            <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100&display=swap" rel="stylesheet">
+            <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300&display=swap" rel="stylesheet">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (jubaan) [inline index.html]: - [ ] Font awesome link is repeated.

- [ ] Roboto font link is repeated.

**Review by jubaan — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang = "en">
+    <head>
+        <title>
+            Apple Store
+        </title>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <meta charset="UTF-8">
+            <link rel="stylesheet" type="text/css" href="stylesheets/style.css">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (expjazz) [inline index.html:10]: - [ ] Remove the extra font-awesome link

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hello :+1: 

You did quite a good job here. However, you have some small problems in order to approve this project:

- [ ] You have linter errors. Kindly fix them
image


- [ ] You have this that was requested in the previous review and still persists:

image

- [ ] You have a small problem in your footer:


| Yours | Original |
|:---:|:---:|
|image |image  |

#### We are almost there. Kindly address those small issues so your project looks even better!

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congratulations Team :clap: :clap: :clap:!!

You may now merge to master.

### [OPTIONAL]

- [ ] Format the HTML file and remove the extra white space. See inline comment.

## See you at the top :rocket: !!

File: index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                        </p>
+                   
+    
+                    
+                        <p>
+                            <a href = "#">Watch the “Your Verse” film and explore the stories</a> 
+                            <i class="fa fa-play-circle watch-icon"></i>
+                        </p>
+                    
+                </section>
```
Comment (jubaan) [inline index.html:50]: - [ ] Remove extra white space

**Pull request merged**